src/Map/doc/index.rst

```diff
@@ -208,21 +208,37 @@ You can add Polylines, which represents a path made by a series of ``Point`` ins
         ),
     ));
 
+Add Circles
+~~~~~~~~~~~
```

Comment:
Not for this PR, but I feel like we should start to link Google Maps/Leaflet documentation for markers, polygones, etc...